changelog/_go-ver-1192.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:change
-core: Bump Go version to 1.23.7.
+core: Bump Go version to 1.24.2.
```

Comment:
This should be in a new file called `changelog/_go-ver-1193.txt` as the changelog entry should match the Vault release version that the Go update is being included in.